android/build.gradle

```diff
@@ -449,9 +442,7 @@ def assertLatestReactNativeWithNewArchitecture = task assertLatestReactNativeWit
     onlyIf { IS_NEW_ARCHITECTURE_ENABLED && REANIMATED_MAJOR_VERSION == 3 && REACT_NATIVE_MINOR_VERSION < 72 }
     doFirst {
         throw new GradleException(
-            "\n[react-native-reanimated] Reanimated " + REANIMATED_VERSION + " supports the New Architecture " +
-            "only on the latest minor release of React Native. Please upgrade to React Native 0.72.0+ " +
-            "or downgrade to an older version of Reanimated v3."
+            "\n[Reanimated] Outdated version of `react-native` for New Architecture. Reanimated " + REANIMATED_VERSION + " supports the New Architecture on React Native 0.72.0+."
```

Comment:
Why did you change this error message? Can we add a link to the Troubleshooting section here?

Reply:
I think originally I wanted to give a link here but then I couldn't decide, definitely we can.